desc/equilibrium/equilibrium.py

```diff
@@ -2015,6 +2017,57 @@ def from_near_axis(
 
         return eq
 
+    @classmethod
+    def from_input_file(cls, path, **kwargs):
+        """Create an Equilibrium from information in a DESC or VMEC input file.
+
+        Parameters
+        ----------
+        path : Path-like or str
+            Path to DESC or VMEC input file.
+        **kwargs : dict, optional
+            keyword arguments to pass to the constructor of the
+            Equilibrium being created.
+
+        Returns
+        -------
+        Equilibrium : Equilibrium
+            Equilibrium generated from the given input file.
+
+        """
+        with warnings.catch_warnings():
+            warnings.simplefilter("ignore")
```

Comment:
won't this ignore possibly important warnings if using a vmec input with incompatible options?

Reply:
we ignore them in the `FourierRZToroidalSurface.from_input_file` because we know we only care about the boundary shape. But for the full equilibrium we also care about profiles, flux, etc which vmec has a lot of options for

Reply:
Good point